_includes/card-operations.md

```diff
@@ -0,0 +1,98 @@
+| `payment`                         | `string`  | {% include field-description-id.md sub_resource=transaction %}                                                                                                                                               |
+| └➔&nbsp;`transaction`                     | `string`  | The transaction object, containing information about the current transaction.                                                                                                                                |
+| └─➔&nbsp;`id`                     | `string`  | {% include field-description-id.md resource=transaction %}                                                                                                                                                   | 
+| └─➔&nbsp;`number`                 | `string`  | The transaction `number`, useful when there's need to reference the transaction in human communication. Not usable for programmatic identification of the transaction, for that `id` should be used instead.                                                                                                                                                               |
+| └➔&nbsp;`payeeReference`          | `string`  | A unique reference for the transaction.                                                                                                                                                                      | 
+| └➔&nbsp;`orderReference`          | `string(50)`  | The order reference should reflect the order reference found in the merchant's systems.                                                                                                                  |                                                             
+| └➔&nbsp;`amount`                  | `integer` | {% include field-description-amount.md %}                                                                                                                                                                    |
+| └➔&nbsp;`tokens`                  | `integer` | List of tokens generated.                                                                                                                                                                                    |
+| └➔&nbsp;`details`                 | `integer` | A human readable and descriptive text of the payment.                                                                                                                                                       | 
+| └─➔&nbsp;`cardBrand`              | `string`  | `Visa`, `MC`, etc. The brand of the card.                                                                                                                                                                    |      
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[hard-break-spaces] Use two spaces for hard line breaks](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5705576)